PIL/Image.py

```diff
@@ -2358,6 +2466,7 @@ def frombuffer(mode, size, data, decoder_name="raw", *args):
 
 
 def fromarray(obj, mode=None):
+    # type: (object, Optional[Mode]) -> Image
```

Comment:
I think this should be a `np.ndarray` type (i.e. `(np.ndarray,Optional[Mode])->Image`).

Reply:
@neiljp I think this is something @wiredfool could resolve. As of now, though, `object` should probably be `Any`.

Reply:
Numpy is certainly an optional dependency for testing, we skip the appropriate tests if it's not installed. This is a small stable interface to numpy, I'm not aware of it changing in the time that we've worked with it. So the exact version isn't terribly important.

Reply:
I've found the following, but haven't yet tested with it:
https://github.com/machinalis/mypy-data/tree/master/numpy-mypy